src/gtclang/Driver/Driver.cpp

```diff
@@ -73,7 +77,7 @@ int Driver::run(const llvm::SmallVectorImpl<const char*>& args) {
   if(context->getOptions().Verbose)
     dawn::Logger::getSingleton().registerLogger(oldLogger);
 
-  return ret;
```

Comment:
Since you have a dependency on boost anyway, what about using boost::optional? (std::optional if you can)

Reply:
The dependency on boost is actually only optional (we only need it *if* we want to compile the code with gridtools). We could however think about switching to C++17